components/SwapWidget.tsx

```diff
@@ -292,13 +292,13 @@ export const SwapWidget = ({ tokenId, tokenSymbol, tokenImage, platforms }: Swap
         </DropdownHeader>
         {isOpen && (
           <DropdownBody>
-            {platforms.ethereum.contractAddress && (
+            {platforms.ethereum && platforms.ethereum.contractAddress && (
```

Comment:
Can make it shorter with:

```suggestion
            {platforms?.ethereum?.contractAddress && (
```

⚠️ haven't tested!! ⚠️ 

Same for the one below

Reply:
Should we test in production?

Reply:
This is not production, so, go for it

Reply:
Tested and committed. Works great, thanks for the suggestion.